addons/account/static/src/js/vat_activity.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+const { Component } = owl;
+
+export class JournalDashboardActivity extends Component {
+    setup() {
+        this.action = useService("action");
+        this.MAX_ACTIVITY_DISPLAY = 5;
+        this.formatData(this.props);
```

Comment:
Shouldn't we put a hook to format the data onPropsWillUpdate as you did somewhere else?

Reply:
That shouldn't be  necessary here, it's not possible to modify `props.value` here.